shopfloor_checkout_package_measurement_mobile/static/wms/src/scenario/checkout_package_measurement.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2022 Camptocamp SA (http://www.camptocamp.com)
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+import {process_registry} from "/shopfloor_mobile_base/static/wms/src/services/process_registry.js";
+
+import {checkout_states} from "/shopfloor_mobile/static/wms/src/scenario/checkout_states.js";
+
+const checkout_scenario = process_registry.get("checkout");
+const template = checkout_scenario.component.template;
+const pos = template.indexOf("</Screen>");
+const new_template =
+    template.substring(0, pos) +
+    `
+
+<div v-if="state_is('package_measurement')">
+
+    <detail-picking
+        :record="state.data.picking"
+        :card_color="utils.colors.color_for('screen_step_done')"
+    />
+    <detail-package
+        :record="state.data.package"
+    />
+
+    <v-form>
+        <v-container>
+            <v-row>
+                <v-text-field
+                    label="Height"
+                    placeholder="Package Height"
+                    v-model="state.data.package.height"
+                ></v-text-field>
+                <v-text-field
+                    label="Length"
+                    placeholder="Package Length"
+                    v-model="state.data.package.length"
+                ></v-text-field>
+                <v-text-field
+                    label="Width"
+                    placeholder="Package Width"
+                    v-model="state.data.package.width"
+                ></v-text-field>
+                <v-text-field
+                    label="Weight"
+                    placeholder="Package Weight"
+                    v-model="state.data.package.weight"
+                ></v-text-field>
+            </v-row>
+        </v-container>
+    </v-form>
+
+    <div class="button-list button-vertical-list full">
+        <v-row align="center">
+            <v-col class="text-center" cols="12">
+                <btn-action action="todo" @click="state.on_done">Done</btn-action>
+            </v-col>
+        </v-row>
+        <v-row align="center">
+            <v-col class="text-center" cols="12">
+                <btn-back />
+            </v-col>
+        </v-row>
+    </div>
+</div>
+
+` +
+    template.substring(pos);
+
+const CheckoutWithPackageMeasurement = process_registry.extend("checkout", {
+    template: new_template,
+    "methods._get_states": function () {
+        let states = checkout_states(this);
+        states["package_measurement"] = {
+            display_info: {
+                title: "Set packaging measurement",
+            },
+            events: {
+                go_back: "on_back",
+                done: "on_done",
```

Comment:
This is useless if you don't trigger the event `done`.
You are using `@click` above -> no need to bind again here AFAIS.

Reply:
Removed